docs/DeveloperGuide.md

```diff
@@ -757,14 +781,111 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes from step 2.
 
-**Use case UCV04: Clear all volunteers in volunteer list**
+**Use case UCEV01: Add volunteer to an event**
 
 **MSS**
 
-1. User clears all volunteers in volunteer list.
-2. The volunteer list becomes empty.
+1. User <u>creates an event (UCE01)</u> with a certain required quantity of roles.
+2. User <u>creates a volunteer (UCV01)</u> with a certain skill.
+3. User requests to add the volunteer to the event.
+4. Volunteer is added to the Event's volunteer list.
```

Comment:
shld this be "iVolunter adds the volunteer to..." to simulate user and system interaction?

Reply:
will fix